test/sequential/sequencerStaking.cancel.test.ts

```diff
@@ -272,6 +286,13 @@ it("GIVEN a sequencer, WHEN <in-correctly> canceling an update AND some pending
     ?.amountBefore.reserved!.sub(
       testUser2.getAsset(GASP_ASSET_ID)?.amountAfter.reserved!,
     );
+  const sequencers = await SequencerStaking.activeSequencers();
+  expect(sequencers.toHuman().Ethereum).not.toContain(
+    testUser1.keyRingPair.address,
+  );
+  expect(sequencers.toHuman().Ethereum).not.toContain(
+    testUser2.keyRingPair.address,
+  );
   expect(testUser1PenaltyValue).bnEqual(
     await SequencerStaking.slashFineAmount(),
   );
```

Comment:
It looks superb, but it seems you are missing the checks for the processed requests.  Both updates with the deposits must be executed, and I can not find any place where u are validating those executions. 
There must be some "RequestExecutedEvents" or perhaps you can validate the balances for the user used in the deposits.

Reply:
here I can see that 
https://github.com/mangata-finance/mangata-e2e/pull/463/files/96ac491ccd2882b33c34fbde98cbbf113023ca9a..46d0f3882783ddfbd4d2d56f4183b6d53258b615#diff-251c1940a8a54baa9fc6c3c0bb65212a09822b46d7ab4a6d214ef14524f288d4R355
after the first deposit, the user must have got some tokens with the same address as his address ( the sequencer address who provided the update ) , and looks that the same must have happened for the second ones,